images/common/install-sambacc-common.sh

```diff
@@ -31,26 +31,48 @@ install_sambacc() {
 
 
     local action=install-from-copr-repo
-    if [ "${#wheels[@]}" -gt 1 ]; then
-        echo "more than one wheel file found"
-        exit 1
-    elif [ "${#wheels[@]}" -eq 1 ]; then
-        action=install-wheel
+    local source_found=0
+    if [ "${#wheels[@]}" -gt 0 ]; then
+        echo "INFO: wheel found"
+        if [ "${#wheels[@]}" -gt 1 ]; then
+            echo "ERROR: more than one wheel file found"
+            exit 1
+        elif [ "${#wheels[@]}" -eq 1  ]; then
+            echo  "sambacc wheel found. Installing wheel."
+            action=install-wheel
+            source_found=1
+        fi
     fi
-
-    if [ "${#rpmfiles[@]}" -gt 1 ]; then
-        echo "more than one sambacc rpm file found"
-        exit 1
-    elif [ "${#rpmfiles[@]}" -eq 1 ]; then
-        action=install-rpm
+    if [ "${#rpmfiles[@]}" -gt 0 ]; then
+        echo "INFO: rpm file found"
+        if [ "${#rpmfiles[@]}" -gt 1 ]; then
+            echo "ERROR: more than one sambacc rpm file found"
+            exit 1
+        elif [ "${#rpmfiles[@]}" -eq 1 ]; then
+            echo "sambacc rpm found. preferring  rpm install."
```

Comment:
Extra space between preferring and rpm: `preferring  rpm`.

Reply:
gone, as is the entire message.